client/reader/liked-stream/empty.jsx

```diff
@@ -45,13 +45,13 @@ class TagEmptyContent extends React.Component {
 					onClick={ this.recordSecondaryAction }
 					href="/discover"
 				>
-					{ this.props.translate( 'Explore Discover' ) }
+					{ this.props.translate( 'Explore' ) }
 				</a>
 			) : null;
 
 		return (
 			<EmptyContent
-				title={ this.props.translate( 'No Likes Yet' ) }
+				title={ this.props.translate( 'No likes yet' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.